test/template_connector/start

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -ex
+
+./stop
+
+docker-compose up -d secretless
```

Comment:
Add comment that they would add the other containers to start.  Or maybe we just do `docker-compose up` to start them all?

Reply:
i have this comment after the line above:
`# This is where you should prepare the container of the platform you are connecting to secretless
`

you think it's not clear enough?

Reply:
@orenbm I'd move the comment above